src/main/java/com/rakovets/course/java/core/practice/concurrency/master_and_worker/Worker.java

```diff
@@ -0,0 +1,35 @@
+package com.rakovets.course.java.core.practice.concurrency.master_and_worker;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Date;
+import java.util.List;
+
+public class Worker  extends Thread{
```

Comment:
"Violation: There isn't separating white spaces for curly brake
Solution: Add white spaces"